src/repairnator-jenkins-plugin/src/main/java/io/jenkins/plugins/main/Config.java

```diff
@@ -92,6 +94,63 @@ public void setTools(String[] tools) {
 		return this.tools;
 	}
 
+	public void setSmtpUsername(String smtpUsername) {
+		stringConfig.put("smtpUsername",smtpUsername);
```

Comment:
if the information is taken from Jenkins, we don't need those setters (duplicating info & state is always to be avoided, it is error prone)

the getters are still useful for encapsulation

Reply:
done :) .